AnkiConnect.py

```diff
@@ -502,11 +509,59 @@ def modelFieldNames(self, modelName):
                 return [field['name'] for field in model['flds']]
 
 
-    def multi(self, actions):
-        response = []
-        for item in actions:
-            response.append(AnkiConnect.handler(ac, item))
-        return response
+    def confForDeck(self, deck):
+        if not deck in self.deckNames():
+            return False
+
+        id = self.collection().decks.id(deck)
+        return self.collection().decks.confForDid(id)
+
+
+    def saveConf(self, conf):
```

Comment:
What is this being used for? It doesn't seem like there are any APIs to actually modify the configuration profiles, so why do we want to save them?